utils/FitUtils.py

```diff
@@ -245,6 +248,22 @@ def _Pol3IntegralNorm(self, x, par):
         fourthTerm = par[3] * (x[0]**3 - 1 / 4. * (self.maxMass**4 - self.minMass**4) / (self.maxMass - self.minMass))
         return firstTerm + secondTerm + thirdTerm + fourthTerm
 
+    def _ExpoPowIntegralNorm(self, x, par):
+        '''
+        Exponential times power law function normalized to its integral.
+ 
+        Parameters
+        ----------
+        - x: function variable
+        - par: function parameters
+        par[0]: normalisation (integral of background)
+        par[1]: expo slope
+        '''
+        mPi = TDatabasePDG.Instance().GetParticle(211).Mass()
```

Comment:
It looks to me that this would be called at each function call. Maybe it is better to add a class member with the pion mass and initialise it in the `__init__` function

Reply:
Done